From e9ce1758917c8a02acdeea91e57d493264fda355 Mon Sep 17 00:00:00 2001
From: ptumaskx <pradyumnax.tumaskar@intel.com>
Date: Fri, 22 May 2026 09:11:48 +0000
Subject: [PATCH] Update RELEASE_PLATFORM_SECURITY_PATCH string

Security_patch_level needs to be updated
When ASB Security patches are integrated

Tracked-On: NIACP3-1351
Signed-off-by: ptumaskx <pradyumnax.tumaskar@intel.com>
---
 flag_values/bp2a/RELEASE_PLATFORM_SECURITY_PATCH.textproto | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/flag_values/bp2a/RELEASE_PLATFORM_SECURITY_PATCH.textproto b/flag_values/bp2a/RELEASE_PLATFORM_SECURITY_PATCH.textproto
index 8d4adba77..4fd5c4044 100644
--- a/flag_values/bp2a/RELEASE_PLATFORM_SECURITY_PATCH.textproto
+++ b/flag_values/bp2a/RELEASE_PLATFORM_SECURITY_PATCH.textproto
@@ -1,4 +1,4 @@
 name: "RELEASE_PLATFORM_SECURITY_PATCH"
 value {
-  string_value: "2025-06-05"
+  string_value: "2026-09-05"
 }
-- 
2.53.0

